### ISSUE TITLE
Swrve 7.0.3 Raising issue with SwrveGeoSDK - Swift

### ISSUE THREAD

**Issue by rana-momenton** [Swrve/swrve-ios-sdk#263]
I am using Swrve and SwrveGeoSDK in one of my library using pod. Yesterday I updated my pod and it updated my Swrve pod to 7.0.3 from 7.0.2. Now I am getting errors in my library as below:

```
**dyld: Symbol not found: _OBJC_CLASS_$_SwrveCommon**
  **Referenced from:** /Users/shahjahanrana/Library/Developer/CoreSimulator/Devices/102EB53D-F0E0-4905-B774-FAECBD3E3126/data/Containers/Bundle/Application/097F24EC-ECF7-4C79-979E-B25A2BF35073/SampleDemoApp.app/Frameworks/SwrveGeoSDK.framework/SwrveGeoSDK

  **Expected in:** /Users/shahjahanrana/Library/Developer/CoreSimulator/Devices/102EB53D-F0E0-4905-B774-FAECBD3E3126/data/Containers/Bundle/Application/097F24EC-ECF7-4C79-979E-B25A2BF35073/SampleDemoApp.app/Frameworks/SwrveSDK.framework/SwrveSDK

 **in** /Users/shahjahanrana/Library/Developer/CoreSimulator/Devices/102EB53D-F0E0-4905-B774-FAECBD3E3126/data/Containers/Bundle/Application/097F24EC-ECF7-4C79-979E-B25A2BF35073/SampleDemoApp.app/Frameworks/SwrveGeoSDK.framework/SwrveGeoSDK
```

I have observed there was a common class being used in SwrveSDK which was also being using in SwrveGeoSDK. Now in 7.0.3, Swrve has introduced a separate library SwrveCommonSDK and moved common files over there so SwrveGeoSDK is unable to find it in SwrveSDK. 

Please can anyone guide me in this reference. I have forced my pods to use 7.0.2 version as of now so that my library can work fine. 
Thanks

**Comment by KevBrodigan** [comment]
@rana-momenton 

Thanks for raising this issue. We are investigating and will fix asap.

**Comment by KevBrodigan** [comment]
This issue has been fixed in SwrveGeo 5.0.1
